test/mocks.go

```diff
@@ -0,0 +1,129 @@
+package test
+
+import (
+	"github.com/sasswart/gin-in-a-can/openapi"
+	"github.com/sasswart/gin-in-a-can/openapi/media"
+	"github.com/sasswart/gin-in-a-can/openapi/operation"
+	"github.com/sasswart/gin-in-a-can/openapi/path"
+	"github.com/sasswart/gin-in-a-can/openapi/request"
+	"github.com/sasswart/gin-in-a-can/openapi/response"
+	"github.com/sasswart/gin-in-a-can/openapi/schema"
+	"github.com/sasswart/gin-in-a-can/tree"
+	"net/http"
+)
+
+var md = tree.Metadata{"package": "testPackage", "some": "metadata"}
+
+func OpenAPITree() *openapi.OpenAPI {
```

Comment:
Remind me why you aren't reading this from a fixture file?

Reply:
These are mocks that are designed to test the renderer without involving the functionality that parses an openAPI spec file